Assets/Code/Sprite/AnimatedSprite/Character/Character.cs

```diff
@@ -4,9 +4,14 @@
 
 public class Character : MonoBehaviour
 {
-    //public int[,] position = new int[1,1];
+    [SerializeField] private int GridPositionX = 0;
+    [SerializeField] private int GridPositionY = 0;
     public SpriteRenderer spriteRenderer;
+    protected PositionGrid grid;
+    public Sprite[] idle;
     private GridCoordinates position;
+    public Sprite[] move;
+    private const float moveAnimationTime = 0.02f;
```

Comment:
Note for the future: We might want to move this moveAnimationTime to a higher level. 

If we end up having custom times for a whole bunch of low level things like this, and then we end up needing to change something, having a centralized location to mess with timing elements might be nice.

Reply:
Agreed. As we expand the animation sets we are using, we should take a look at pushing some things like this up into a parent class - perhaps AnimatedSprite?